hack/versions-management/tag_reconciler.sh

```diff
@@ -6,6 +6,8 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 source "$SCRIPT_DIR/github_auth.sh"
 
+pip install -r "$SCRIPT_DIR/requirements.txt"
```

Comment:
Wouldn't this be best in the makefile, and add it as dependency in the targets that might need it?